src/veh_interact.cpp

```diff
@@ -847,15 +847,29 @@ bool veh_interact::update_part_requirements()
         ok = false;
     }
 
+    std::string str_suffix;
+    int lift_strength = player_character.get_lift_str();
+    int total_lift_strength = lift_strength + player_character.get_lift_assist();
+    int total_base_strength = player_character.get_str() + player_character.get_lift_assist();
+
+    if( player_character.has_trait( trait_id( "STRONGBACK" ) ) && total_lift_strength >= str &&
+        total_base_strength < str ) {
+        str_suffix = string_format( _( "(Strong Back helped, giving +%d strength)\n" ),
```

Comment:
I think `\n` is not necessary here.

Reply:
i ll take a look, but i think without it any automatic linebreak ends up at the end of the description header.

Reply:
I noticed there was a display bug so I made #47121

Reply:
ah thank you for the fix :)